components/requirement-list.jsx

```diff
@@ -0,0 +1,47 @@
+var React = require('react');
+
+var RequirementRow = React.createClass({
+    propTypes: {
+        text        : React.PropTypes.string.isRequired,
+        icon        : React.PropTypes.string
+    },
+    render: function () {
+        var icon = this.props.icon ? this.props.icon : 'fa fa-check';
+        return (
+            <li>
+                <div className="icon-wrapper">
+                    <span className={icon}></span>
+                </div>
+                <div className="text">
+                    { this.props.text }
+                </div>
+            </li>
+        );
+    }
+});
+
+
+var RequirementsList = React.createClass({
+    propTypes: {
+        list        : React.PropTypes.array.isRequired,
+        className   : React.PropTypes.string,
+        icon        : React.PropTypes.string
+    },
+    render: function () {
+        var className = this.props.className ? 'list-unstyled requirements-list' + this.props.className : 'list-unstyled requirements-list';
+        var icon = this.props.icon ? this.props.icon : 'fa fa-check';
+        var i = 0;
+        var listItems = this.props.list.map(function(item){
+            i++;
+            return <RequirementRow key={i} icon={icon} text={item} />
```

Comment:
`key` should not be dependent on the list position, but should be based on the item itself. It's used to determine whether the dom diff needs to delete/insert, or simply move things (much faster).

That said, `map` and `forEach` etc can take a second argument so you don't need an external counter:
```
['a','b','c','d'].map(function(letter, position) {
  console.log(position + ": " + letter);
});
```
generates
```
0: a
1: b
2: c
3: d
```